### DISCUSSION TITLE
headless doesnt seem to work now.

### DISCUSSION THREAD

**Discussion by cuppjj** [ultrafunkamsterdam/undetected-chromedriver#1253] [General]
import undetected_chromedriver as uc
driver = uc.Chrome(headless=True)
driver.get('https://nowsecure.nl')

This used to work until today. Anyone else having issues?

**Comment by cuppjj** [comment]
Got it to work with this 
driver = uc.Chrome(enable_cdp_events=True, headless=True, version_main=112)

**Reply by silverxd20** [reply]
it works like a charm, thanks a lot.
